inst/components/scss/page_fillable.scss

```diff
@@ -10,7 +10,63 @@
 }
 
 @include media-breakpoint-down(sm) {
-  .bslib-page-fill {
-    height: var(--bslib-page-fill-mobile-height, auto);
+  .bslib-flow-mobile {
+    > .html-fill-item {
+      flex: 0 0 auto;
+    }
+
+    &.bslib-page-sidebar,
+    &.bslib-page-navbar.has-page-sidebar {
+      // Keep filling behavior of outer container
+      > .html-fill-item {
+        flex: 1 1 auto;
+      }
+    }
+
+    &.bslib-page-sidebar,
+    &.bslib-page-navbar.has-page-sidebar > .html-fill-container {
+      > .bslib-sidebar-layout {
+        // Switch to flow layout in main area of the sidebar container
+        > .main > .html-fill-item {
+          flex: 0 0 auto;
+        }
+
+        &:not(.sidebar-right) {
+          > .main {
+            padding-left: var(--_padding-icon);
+            padding-top: var(--_padding);
+          }
+          > .collapse-toggle {
+            left: calc(var(--_icon-size) / 2);
+          }
+        }
+        &.sidebar-right {
+          > .main {
+            padding-right: var(--_padding-icon);
+          }
+          > .collapse-toggle {
+            right: calc(var(--_icon-size) / 2);
+          }
+        }
+      }
+    }
+  }
+}
+
+@include media-breakpoint-down(sm) {
+  .bslib-flow-sm > .html-fill-item {
+    flex: 0 0 auto;
+  }
+}
+
+@include media-breakpoint-down(md) {
+  .bslib-flow-md > .html-fill-item {
+    flex: 0 0 auto;
+  }
+}
+
+@include media-breakpoint-down(lg) {
+  .bslib-flow-lg > .html-fill-item {
+    flex: 0 0 auto;
```

Comment:
Unless we have immediate plans to use and document these, I think I'd prefer to not commit to supporting them, especially right before a release. It kinda feels like we might want to revisit this concept but with height-related container query?

```suggestion
```

Reply:
That makes sense. I think the `.bslib-flow-mobile` class is enough for this release and reasonably covers most of the intent here.